watchguard_firebox/assets/dashboards/watchguard_firebox_networking_events.json

```diff
@@ -0,0 +1,757 @@
+{
+    "title": "WatchGuard Firebox - Networking Events",
+    "description": "Networking events are generated for traffic related to the connections through your Firebox. This can include events related to interface activity, dynamic routing and PPPoE connections.",
```

Comment:
```suggestion
    "description": "Networking events are generated for traffic related to the connections through your Firebox. This can include events related to interface activity, dynamic routing, and PPPoE connections.",
```